### DISCUSSION TITLE
Mass label contacts

### DISCUSSION THREAD

**Discussion by arturritzel** [chatwoot/chatwoot#9915] [Chatwoot APIs] [answered]
I want to mass label hundreds of contacts but couldn't find a way to do it. Maybe using an specific API endpoint?
Is it possible to import contacts with labels? When I export contacts I see there are fields for id, name, email, phone number, but couldn't find a way to import contacts with labels. Also, how should my contacts csv look like when importing contacts? I assume I should not define an id for contacts, for example.

**Comment by Paul181516** [comment]
Did you find a solution for this?

**Reply by arturritzel** [reply, marked as answer]
Sojan (Chatwoot's CTO) answered me on discord:
![image](https://github.com/user-attachments/assets/46476ed6-ad34-4d11-85ce-03e704d15510)

**Comment by bretoreta** [comment]
This is just so repetetive, imagine having thousands of contacts and having to label only 15 at a time.
